test/sharness/t0020-init.sh

```diff
@@ -44,4 +44,16 @@ test_expect_success "ipfs init output looks good" '
 	test_cmp expected actual_init
 '
 
+test_init_ipfs
+
+test_launch_ipfs_daemon
+
+test_expect_failure "ipfs init should not run while daemon is running" '
+	ipfs init 2> daemon_running_err &&
+	expect="Error: ipfs daemon is running. please stop it to run this command" &&
+	cat daemon_running_err | grep $expect
```

Comment:
i think i had that somewhere and @chriscool preferred the `cat <file> | grep <pattern>` approach.